src/components/hooks/useInstanceList.jsx

```diff
@@ -19,7 +20,18 @@ export default function useInstanceList() {
 		}
 		if (list?.pages) {
 			let dataMap = []
-			return [...dataMap.concat(...list.pages.map((page) => page.pagination))].sort(_compareWidgets)
+			return [
+				...dataMap.concat(
+					...list.pages.map(page => page.pagination.map(instance => {
+						// adding an 'img' property to widget instance objects for continued
+						//  compatibility with any downstream LTIs using the widget picker
+						return {
+							...instance,
+							img: iconUrl(BASE_URL + 'widget/', instance.widget.dir, 60)
```

Comment:
This works fine, my only suggestion might be to use 275 instead of 60 for the icon size. We're also making use of `iconUrl` in the `my-widgets-instance-card` component, and by using 275 we can effectively replace both `iconUrl` references with a reference to the `img` property of the instance. That line is here: https://github.com/ucfopen/Materia/blob/issue/support-dashboard-in-react/src/components/my-widgets-instance-card.jsx#L30

I also noticed this does indeed fix the icon not being displayed during the progress bar view of the `select-item` component, but it's missing some styles to size and position it correctly. I'm happy to fix that after the merge if you prefer.